streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -500,7 +472,7 @@ public Bytes peekNextKey() {
         }
     }
 
-    private class RocksDBRangeIterator extends RocksDbIterator {
+    static class RocksDBRangeIterator extends RocksDbIterator {
```

Comment:
Ok. This is an actual change. Need to make this class `static` to be reused in new `RocksDBTimestampedStore` class. For this reason, we need to pass in `openIterators` in the constructor now.

Reply:
Ack